.gitignore

```diff
@@ -115,3 +115,8 @@ SECRET
 # substra-backend files
 docker/docker-compose-dynamic.yaml
 **/medias/*
+
+
+# charts tmp files
+charts/substra-backend/tmpcharts/*
```

Comment:
@ClementGautier In a working setup tmpcharts should be remove automatically ?
If not, nice catch 👍

Reply:
Yes tmpcharts are supposed to be cleaned up by skaffold unless you kill it violently. I've no problem adding it to the gitignore tho.